packages/vscode-extension/src/notifications/AlertsHandler.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Notification, NotificationSeverity } from "@kogito-tooling/notifications/dist/core";
+import { WorkspaceApi } from "@kogito-tooling/workspace/dist/api";
+import * as vscode from "vscode";
+import { NotificationHandler } from "./NotificationHandler";
+
+class AlertsHandler implements NotificationHandler {
```

Comment:
```suggestion
export class AlertsHandler implements NotificationHandler {
```

You can directly export this class and remove line 65.